### DISCUSSION TITLE
Renaming Label

### DISCUSSION THREAD

**Discussion by bm55** [photoprism/photoprism#1171] [Q&A] [answered]
I had "cup" and "Beverage" labels. I renamed the "cup" to "Beverage" and ran the index. Now I get 2 labels both called "Beverage" with same images inside. Do I need to manually delete one of the labels?

**Comment by graciousgrey** [comment, marked as answer]
Hey, that happens because we have beverage as a label  as well. So before you renamed the label there have been "cup" and "beverage" as label and after you renamed cup you have "beverage" twice one is the original "beverage" and the other one is the renamed "cup"
